Parent/Parent/ParentAppDelegate.swift

```diff
@@ -106,9 +108,14 @@ class ParentAppDelegate: UIResponder, UIApplicationDelegate {
         Analytics.shared.logSession(session)
         getPreferences { userProfile in performUIUpdate {
             LocalizationManager.localizeForApp(UIApplication.shared, locale: userProfile.locale) {
-                GetBrandVariables().fetch(environment: self.environment) { [weak self] _, _, _ in performUIUpdate {
-                    self?.showRootView()
-                }}
+                ReactiveStore(useCase: GetBrandVariables())
+                    .getEntities()
+                    .receive(on: RunLoop.main)
+                    .sink(receiveCompletion: { _ in }) { [weak self] brandVars in
```

Comment:
nit: fist parameter could be omitted, unless I'm mistaken
(same for the other appdelegates)